src/open_inwoner/openzaak/tests/factories.py

```diff
@@ -1,5 +1,5 @@
 import factory
-from zgw_consumers.constants import CertificateTypes
+from simple_certmanager.constants import CertificateTypes
 from zgw_consumers.models import Certificate, Service
```

Comment:
We should update the factory as well.

Although it is interesting this test apparently still passes on this PR even though we save another model. I guess because the model is a copy? https://github.com/maykinmedia/open-inwoner/blob/2ae82764306fb19ab748ceb99db055679d52b87f/src/open_inwoner/openzaak/tests/test_documents.py#L308-L318

Reply:
yeah, afaik `zgw_consumers.Service` model still exists but it's a proxy for `simple_certmanager.Service` model

Reply:
I've changed import to use "real" model :)

Reply:
Cool, let's resolve!